microovn/frr/bgp/redirect.go

```diff
@@ -137,10 +138,19 @@ func getVrfName(tableID string) string {
 	return fmt.Sprintf("ovnvrf%s", tableID)
 }
 
+func getBgpVethName(externalIface string) string {
+	return fmt.Sprintf("v-%s-bgp", externalIface)
```

Comment:
Interface name real estate is scarce:
https://github.com/torvalds/linux/blob/07d9df80082b8d1f37e05658371b087cb6738770/include/uapi/linux/if.h#L33

Can you think of a way to make this shorter?